chroma-manager/tests/integration/utils/test_blockdevices/test_blockdevice_lvm.py

```diff
@@ -23,6 +23,7 @@ def prepare_device_commands(self):
         # FIXME: the use of --yes in the {vg,lv}create commands is a work-around for #500
         # and should be reverted when #500 is fixed
         return [
+            "wipefs -a %s" % self._device_path,
```

Comment:
Why is this now needed when it wasn't before?

Reply:
This is a good question.  But the `wipefs` is probably still a good measure if we are unable to guarantee the consistency of the device before this test.